src/screens/auth/VerifyEmail/VerifyEmail.tsx

```diff
@@ -4,26 +4,50 @@ import { View } from 'react-native';
 import { Text } from 'src/components';
 import { ScreenTemplate } from 'src/components/templates';
 import { RootStackParamList } from 'src/navigation';
+import { useAppDispatch } from 'src/store';
+import { AsyncThunks } from 'src/store/thunks';
 
 import styles from './VerifyEmail.style';
 
-const VerifyEmail = () => {
+const VerifyEmail = ({ route }: any) => {
```

Comment:
Avoid usage of `any` type. The correct types should cover everything.

In this case, you can import the `Route` type from `@react-navigation/native` and apply it here.